doc/rados/configuration/mon-config-ref.rst

```diff
@@ -420,6 +420,14 @@ by setting it in the ``[mon]`` section of the configuration file.
 :Default: ``0``
 
 
+``mon warn on pool replica size of one``
```

Comment:
can we change this also to be `` mon warn on pool no redundancy``?  sorry i should have said that earlier!

Reply:
Sure, I have changed it now.